.github/workflows/tests.yml

```diff
@@ -8,6 +8,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+      - name: Wait for dpkg lock
+        run: |
+          while sudo fuser /var/lib/dpkg/lock-frontend >/dev/null 2>&1; do
```

Comment:
Is there a possibility that this could block forever?  Perhaps it's better to fail fast...